CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Slice attributes of `attribute` package are now comparable based on their value, not instance. (#3108 #3252)
 - Prometheus exporter will now cumulatively sum histogram buckets. (#3281)
 - Export the sum of each histogram datapoint uniquely with the `go.opentelemetry.io/otel/exporters/otlpmetric` exporters. (#3284, #3293)
+- Getting-started guide will now have the right list of import packages in main.go to run through the example successfully. (#3354)
```

Comment:
I'm not sure website_docs fixes require a changelog entry.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/3354/commits/352dcd7bc260fba14178c5059f468a80ecb70c26

Reply:
Thanks for changing that. I was not aware doc change doesn't need to go into changelog based on [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md). Any written rules on this I can refer for future?